docker-compose.yaml

```diff
@@ -31,33 +28,16 @@ services:
       POSTGRES_USER: "${POSTGRES_USER}"
       POSTGRES_HOST_AUTH_METHOD: "md5"
       PGDATA: /var/lib/postgresql/data/pgdata
-    networks:
-      main:
-        ipv4_address: 172.25.0.12
 
   frontend:
-    image: node:latest
     volumes:
-      - node_modules:/app/node_modules
       - ./config.json:/app/config.json:ro
     ports:
-      - 127.0.0.1:5559:3000
+      - 5559:3000
     container_name: "mystbin-frontend"
     build: ./mystbin/frontend
     depends_on:
       - api
-    networks:
-      main:
-        ipv4_address: 172.25.0.13
 
 volumes:
-  postgresdata:
-  node_modules:
-
-
-networks:
-  main:
-    ipam:
-      driver: default
-      config:
-        - subnet: 172.25.0.0/16
```

Comment:
I cannot recall why I had explicit networking, if I recall I'll edit this back in but thank you.